site/src/app/v2/test-eligibilite/components/eligibility-test-forms/EligibilityTestForms.tsx

```diff
@@ -0,0 +1,80 @@
+import { useState } from 'react';
+import StepOneForm from '../step-one-form/StepOneForm';
+import YoungCafForm from '../step-two-forms/YoungCafForm';
+import { ConfirmResponseBody, SearchResponseBody } from 'types/EligibilityTest';
+import YoungMsaForm from '../step-two-forms/YoungMsaForm';
+import AahCafForm from '../step-two-forms/AahCafForm';
+import QrCodeVerdict from '../qrcode-verdict/QrCOdeVerdict';
+import AahMsaForm from '../step-two-forms/AahMsaForm';
+import FullNegativeVerdictPanel from '@/app/components/verdictPanel/FullNegativeVerdictPanel';
+
+const EligibilityTestForms = () => {
+  const [eligibilityData, setEligibilityData] = useState<SearchResponseBody | null>(null);
+  const [pspCodeData, setpspCodeData] = useState<ConfirmResponseBody | null>(null);
+
+  return (
+    <div>
+      <StepOneForm
+        onDataRecieved={(data: SearchResponseBody) => {
+          setEligibilityData(data);
+          setpspCodeData(null);
+        }}
+      />
+
+      {eligibilityData &&
+        eligibilityData.length > 0 &&
+        eligibilityData[0].situation === 'jeune' &&
+        eligibilityData[0].organisme === 'CAF' && (
+          <YoungCafForm
+            eligibilityDataItem={eligibilityData[0]}
+            onDataRecieved={(data: ConfirmResponseBody) => setpspCodeData(data)}
+          />
+        )}
```

Comment:
Tu pourrais utiliser "get" de lodash (package à installer à priori) pour rendre les conditions plus concises, mais pas nécessaire dans cette PR
```suggestion
      {
      get(eligibilityData, '[0].situation', null) === 'jeune' && 
      get(eligibilityData, '[0].organisme', null) === 'CAF' && (
          <YoungCafForm
            eligibilityDataItem={eligibilityData[0]}
            onDataRecieved={(data: ConfirmResponseBody) => setpspCodeData(data)}
          />
        )}
```

Reply:
Ou tout simplement utiliser l'optional chaining https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

eligibilityData?.[0]?.situation === 'jeune' etc.

Reply:
lodash n'est pas installé. (et j'ai pas cherché à implifier). Donc oui, pour plus tard